src/index.html

```diff
@@ -15,8 +19,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          class="logo-img"
+          src="images/logo.png"
+          alt="moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link"
+              href="#"
```

Comment:
The `href` attribute is set to '#', which is a placeholder. Ensure this link points to a meaningful destination or is handled appropriately in JavaScript.